yb-voyager/cmd/importData.go

```diff
@@ -83,6 +84,24 @@ type ImportFileTask struct {
 	TableName string
 }
 
+func quoteTableNameIfRequired() {
+	if tconf.TargetDBType != ORACLE {
+		return
+	}
+	for _, fileEntry := range dataFileDescriptor.DataFileList {
+		if sqlname.IsQuoted(fileEntry.TableName) {
+			return
+		}
+		if sqlname.IsReservedKeyword(fileEntry.TableName) ||
```

Comment:
`sqlname.IsReservedKeyword()` is a YB/PG specific check -- not applicable to Oracle.